mgradm/cmd/distro/cp.go

```diff
@@ -98,28 +74,98 @@ func distroCp(
 			srcdir,
 		}
 		if out, err := utils.RunCmdOutput(zerolog.DebugLevel, "/usr/bin/sudo", mountCmd...); err != nil {
-			log.Debug().Msgf("Error mounting ISO image: '%s'", out)
-			return errors.New(L("unable to mount ISO image. Mount manually and try again"))
+			log.Debug().Msgf(L("Error mounting ISO image: '%s'"), out)
+			return "", needremove, fmt.Errorf(L("unable to mount iso file: %s"), out)
```

Comment:
```suggestion
			return "", needremove, fmt.Errorf(L("Unable to mount ISO image: %s"), out)
```

Reply:
I was intentionally keeping error messages lowercase as I was yelled at by vscodium linter that error messages should not be capitalized.

Reply:
My bad, the suggestion should have been:

```suggestion
			return "", needremove, fmt.Errorf(L("unable to mount ISO image: %s"), out)
```